spec.html

```diff
@@ -32086,27 +32086,127 @@ <h1>Week Day</h1>
         <p>A weekday value of *+0*<sub>𝔽</sub> specifies Sunday; *1*<sub>𝔽</sub> specifies Monday; *2*<sub>𝔽</sub> specifies Tuesday; *3*<sub>𝔽</sub> specifies Wednesday; *4*<sub>𝔽</sub> specifies Thursday; *5*<sub>𝔽</sub> specifies Friday; and *6*<sub>𝔽</sub> specifies Saturday. Note that <emu-eqn>WeekDay(*+0*<sub>𝔽</sub>) = *4*<sub>𝔽</sub></emu-eqn>, corresponding to Thursday, 1 January 1970.</p>
       </emu-clause>
 
-      <emu-clause id="sec-local-time-zone-adjustment" type="implementation-defined abstract operation">
+      <emu-clause id="sec-roundtowardszero" type="abstract operation">
         <h1>
-          LocalTZA (
-            _t_: a Number,
-            _isUTC_: a Boolean,
-          ): an integral Number
+          RoundTowardsZero (
+            _x_: a mathematical value,
+          ): an integer
+        </h1>
+        <dl class="header">
+          <dt>description</dt>
+          <dd>The returned integer is the integral part of _x_, with any fractional part removed.</dd>
+        </dl>
+        <emu-alg>
+          1. Return the mathematical value that is the same sign as _x_ and whose magnitude is floor(abs(_x_)).
+        </emu-alg>
+      </emu-clause>
+
+      <emu-clause id="sec-getepochfromisoparts" type="abstract operation">
+        <h1>
+          GetEpochFromISOParts (
+            _year_: an integer,
+            _month_: an integer between 1 and 12 inclusive,
+            _day_: an integer between 1 and 31 inclusive,
+            _hour_: an integer between 0 and 23 inclusive,
+            _minute_: an integer between 0 and 59 inclusive,
+            _second_: an integer between 0 and 59 inclusive,
+            _millisecond_: an integer between 0 and 999 inclusive,
+            _microsecond_: an integer between 0 and 999 inclusive,
+            _nanosecond_: an integer between 0 and 999 inclusive,
+          ): a BigInt
         </h1>
         <dl class="header">
           <dt>description</dt>
-          <dd>Its return value represents the local time zone adjustment, or offset, in milliseconds. The local political rules for standard time and daylight saving time in effect at _t_ should be used to determine the result in the way specified in this section.</dd>
+          <dd>The returned value represents a number of nanoseconds since the Unix epoch in UTC that corresponds to the given ISO 8601 calendar date and wall-clock time in UTC.</dd>
```

Comment:
I'd hesitate to have a cross-reference to that whole section, because it might imply that the value here is a time value (in 𝔽). I propose making "epoch" into a `<dfn>` in that section, and having it automatically link here.